cmd/manager/main.go

```diff
@@ -25,12 +26,18 @@ import (
 	machineactuator "sigs.k8s.io/cluster-api-provider-aws/pkg/actuators/machine"
 	"sigs.k8s.io/cluster-api-provider-aws/pkg/apis/awsproviderconfig/v1beta1"
 	awsclient "sigs.k8s.io/cluster-api-provider-aws/pkg/client"
+	"sigs.k8s.io/cluster-api-provider-aws/pkg/version"
 	"sigs.k8s.io/controller-runtime/pkg/client/config"
 	"sigs.k8s.io/controller-runtime/pkg/manager"
 	"sigs.k8s.io/controller-runtime/pkg/runtime/signals"
 )
 
 func main() {
+	if len(os.Args) > 1 && os.Args[1] == "version" {
+		fmt.Println(version.String)
```

Comment:
why not `glog.fatal(...)` instead of fmt?

Reply:
Which one do you prefer if you execute `manager version`, think this is more a command for humans.

`ClusterAPIProviderAWS v0.2.0-64-g0afb3b90-dirty`

`F0315 15:14:05.721565   95782 main.go:48] ClusterAPIProviderAWS v0.2.0-64-g0afb3b90-dirty`

Reply:
Strong +1 for just using `fmt.Println()` here.

Reply:
Makes sense :+1: